core/lib/protobuf_config/src/proto/config/da_client.proto

```diff
@@ -36,9 +36,32 @@ message CelestiaConfig {
   optional uint64 timeout_ms = 4;
 }
 
+message MemStoreConfig {
+  optional uint64 max_blob_size_bytes = 3;
+  optional uint64 blob_expiration = 4;
+  optional uint64 get_latency = 5;
+  optional uint64 put_latency = 6;
+}
+
+message DisperserConfig {
+  optional string disperser_rpc = 3;
+  optional int32 eth_confirmation_depth = 4;
+  optional string eigenda_eth_rpc = 5;
+  optional string eigenda_svc_manager_address = 6;
+  optional uint32 blob_size_limit = 7;
+  optional uint64 status_query_timeout = 8;
+  optional uint64 status_query_interval = 9;
+  optional bool wait_for_finalization = 10;
+  optional bool authenticated = 11;
+  optional bool verify_cert = 12;
+  optional string path_to_points = 13;
+}
+
 message EigenConfig {
-  optional string rpc_node_url = 1;
-  optional uint64 inclusion_polling_interval_ms = 2;
+  oneof config {
```

Comment:
breaking proto change, please use `reserved` instruction to avoid it

Reply:
fixed here https://github.com/lambdaclass/zksync-era/pull/343